lib/Gitlab/Api/AbstractApi.php

```diff
@@ -1,6 +1,11 @@
 <?php namespace Gitlab\Api;
 
 use Gitlab\Client;
+use Gitlab\HttpClient\Message\ResponseMediator;
+use function GuzzleHttp\Psr7\stream_for;
```

Comment:
You need to remove this line. It will couple us to Guzzle. You should use the `StreamFactoryDiscovery` instead.